hack/jenkins/scripts/gather-logs.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+set -x
+set -euo pipefail
+
+export IDENT="${IDENT:-"${HOME}/.ssh/id_rsa"}"
+
+cd ${DIR}/../../terraform-quickstart/
+
+# TODO: unclear why ssh-agent isn't still running from tqs-up.sh...
+if [ -z "${SSH_AUTH_SOCK:-}" ] ; then
```

Comment:
Is the next task to move the ssh-agent to the tqs-up.sh?

Reply:
It already is in `tqs-up.sh`. I don't trap/kill it, so it ought to still be running, but `ps aux | grep ssh` clearly shows that it isn't, so I just start it up again.

I probably will leave this workaround for the time being. My intuition is that it's another quirk of how the kubernetes-plugin works in Jenkins, and I feel it could take a lot of time to chase down with very little net benefit.

Reply:
Thanks for the clarification

Reply:
are we using the ssh agent plugin in jenkins? it tears down the agent after the 'build' step for some reason.

Reply:
No, in this case, I manually start the ssh-agent inside `tqs-up.sh` which should start a process and drop an `env` file. I checked to see if maybe it was just that the environment was getting reset between `stage(s)` but there was no relevant ssh process running at this point anyway, so somehow that process was getting cleaned up.

We could potentially consider using the sshAgent pipeline extension, but I'm reluctant to add back more plugins. Every single plugin, so far, has come with at least one sharp edge.

Reply:
Agreed... +1 to use the least amount of plugins